akka-http/src/main/scala/akka/http/scaladsl/server/directives/BasicDirectives.scala

```diff
@@ -402,6 +402,8 @@ trait BasicDirectives {
           throw IllegalRequestException(
             StatusCodes.RequestTimeout,
             ErrorInfo(s"Request timed out after $timeout while waiting for entity data", "Consider increasing the timeout for toStrict"))
+        case EntityStreamException(info) =>
+          throw IllegalRequestException(StatusCodes.BadRequest, info)
```

Comment:
`info` is already split into a `summary` that may be public and a `detail` that should not be included in the response by default, so that's :+1:

Reply:
Good idea - detail added.